langchain/schema.py

```diff
@@ -182,9 +182,19 @@ class LLMResult(BaseModel):
     each input could have multiple generations."""
     llm_output: Optional[dict] = None
     """For arbitrary LLM provider specific output."""
-    run: Optional[RunInfo] = None
+    run: Optional[List[RunInfo]] = None
     """Run metadata."""
 
+    def flatten(self) -> List[LLMResult]:
+        """Flatten generations into a single list."""
+        return [
+            LLMResult(
+                generations=[gen],
+                llm_output=self.llm_output,
```

Comment:
this will produce widely wrong token counts in lcp

Reply:
yup will fix